### ISSUE TITLE
[Suggestion] Window Resizing Split Horizontally

### ISSUE THREAD

**Issue by eeyun** [budgie-desktop/budgie-desktop#975]
TL;DR being able to split windows both vertically and horizontally would be a wonderfully useful feature!

I'm a recent convert to budgie and I come from a tiling window manager. I was really happy that `Super+Left` and `Super+Right` preset for halving my workspace vertically with two windows. But, I find myself very frequently wishing I could split horizontally as well. If there were presets for sending a window top and bottom that would basically solve any remaining pain points for the transition from a tiling WM and allow users a quick and easy way to display 3 or four windows at once without requiring switching over to a separate workspace.

By and large, this need is because I'm prettttty bad a programming and frequently need my IDE on screen, zeal docs, and slack or IRC all at once.

**Comment by ikeydoherty** [comment]
We can't do this in Mutter, sorry.

**Comment by eeyun** [comment]
Understood, perhaps I can open an issue on Mutter or research whether it's possible to implement there. Otherwise, I'm going to close this out :) . If I have any useful updates on this at any point I'll re-open.

**Comment by Jacob-Vlijm** [comment]
Hi @eeyun, would an edited version of this script help you out for now? https://askubuntu.com/questions/576427/can-i-set-a-keybinding-to-resize-a-window-to-half-its-size-in-a-chosen-directio.

**Comment by eeyun** [comment]
@Jacob-Vlijm Yes, it totally does! I'm basically doing this with my own scripts right now and as a workaround it is reasonable. But first class is always better than bolt on to be sure.
